raster-test/src/test/scala/geotrellis/raster/io/geotiff/GeoTiffMultiBandTileSpec.scala

```diff
@@ -12,11 +12,11 @@ import geotrellis.testkit._
 
 import org.scalatest._
 
-class GeoTiffMultiBandTileSpec extends FunSpec
+class XGeoTiffMultiBandTileSpec extends FunSpec
```

Comment:
I just wanted to give this one a unique prefix compared to the other tests which match the pattern `*GeoTiffMultiBandTileSpec` so that I could run this one without the others.  I don't know if there is an easier way to do this.  This change does not need to be permanent (can be removed before merging).